Networking page update (#12268)
gravitational/teleport: docs/pages/setup/reference/networking.mdx

| 3023 | Proxy | SSH port clients connect to. The Proxy Service will forward this connection to port `#3022` on the destination Node. |
| 3024 | Proxy | SSH port used to create "reverse SSH tunnels" from behind-firewall environments into a trusted proxy server. |
| 3025 | Auth | SSH port used by the Auth Service to serve its API to other Nodes in a cluster. |

> | 3080 | Proxy | HTTPS connection to authenticate `tsh` users into the cluster. Typically set to 443 instead of the default 3080. The same connection is used to serve a Web UI. In addition to HTTPS, WebSocket traffic, wss://..., is used on this port. |

Reviewer: > Typically set to 443 instead of the default 3080

Is it though? Binding to a low port requires running as root which I don't know if many users run proxies as root (I may be mistaken). What you probably meant is that typically Proxy is deployed behind a load balancer that forwards 443 to 3080 on proxy? So the listen port may still be 3080 but the public advertise port is 443.

Author: updated @r0mant

| Port | Service | Description |
| - | - | - |
| 3023 | Proxy | SSH port clients connect to. The Proxy Service will forward this connection to port `#3022` on the destination Node. |
| 3024 | Proxy | SSH port used to create "reverse SSH tunnels" from behind-firewall environments into a trusted proxy server. |

> | 443 | Proxy | HTTPS connection to authenticate `tsh` users into the cluster. The same connection is used to serve a Web UI. WebSocket traffic is used as part of the Web interaction. |

Reviewer 2: ```suggestion
| 443 | Proxy | HTTPS connection to authenticate `tsh` users into the cluster. The same connection is used to serve a Web UI and any WebSocket traffic.|
```
To make this more concise (assuming the meaning is the same)
